client.js

```diff
@@ -100,75 +112,94 @@ RingpopClient.prototype.destroy = function destroy(callback) {
     if (this.isChannelOwner) {
         this.tchannel.close(callback);
     }
+
+    this.timers.clearTimeout(this.wedgedTimer);
+    this.isDestroyed = true;
 };
 
 /* jshint maxparams: 5 */
 RingpopClient.prototype._request = function _request(opts, endpoint, head, body, callback) {
     var self = this;
 
-    if (this.subChannel && this.subChannel.destroyed) {
-        process.nextTick(function onTick() {
-            callback(ChannelDestroyedError({
-                endpoint: endpoint,
-                channelType: 'subChannel'
-            }));
-        });
-        return;
+    if (!this.wedgedTimer) {
+        this._scheduleWedgedTimer();
     }
 
-    if (this.subChannel &&
-            this.subChannel.topChannel &&
-            this.subChannel.topChannel.destroyed) {
+    var request = new ClientRequest(this, opts, endpoint, head, body, onSend);
+
+    // Evaluate if Ringpop should start applying backpressure on itself.
+    // Allow only a certain amount of potentially wedged requests to exist.
+    // It is highly unlikely that this'll ever be triggered, but it's best
+    // not to issue an unbounded number of wedged requests.
```

Comment:
this comment is slightly confusing to me -- we don't issue wedged requests, so much as issue requests and monitor for wedgies, right? :)